### PR TITLE
Migrate to external neuro-flow actions

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

**Comment by yartem** [comment]
Hint to fix windows:
1. https://github.com/neuro-inc/neuro-extras/blob/7b2de8ee5d5324e23f35352f214a94dab9b7caa3/.github/workflows/cd.yaml#L164
2. (optional) `--color=no` https://github.com/neuro-inc/neuro-extras/blob/7b2de8ee5d5324e23f35352f214a94dab9b7caa3/.github/workflows/cd.yaml#L208

**Review by asvetlov — COMMENTED** [review]
I suggest using curly brackets

File: {{cookiecutter.project_slug}}/.neuro/live.yml [hunk 1]
```diff
@@ -88,46 +83,23 @@ jobs:
...
-    env:
-      PYTHONPATH: $[[ volumes.code.mount ]]
-    cmd: >-
-      jupyter $[[ multi.args ]]
-        --no-browser
-        --ip=0.0.0.0
-        --allow-root
-        --NotebookApp.token=
-        --notebook-dir=$[[ volumes.notebooks.mount ]]
+    action: gh:neuro-actions/jupyter@master
```
Comment (yartem) [inline {{cookiecutter.project_slug}}/.neuro/live.yml:86]: where is the action's repo located? https://github.com/neuro-inc/neuro-actions/jupyter -> 404

Comment (anayden) [reply {{cookiecutter.project_slug}}/.neuro/live.yml:86]: We've got a separate GH org for neuro-flow actions: https://github.com/neuro-actions/jupyter

**Review by YevheniiSemendiak — COMMENTED** [review]
@anayden, thanks for refactoring and publishing components as GH actions, they definitely will be handy for our users 👍 

2 general remarks about this:
- Is it possible to add small READMEs to those GH actions? At least one-sentence description and list of assumptions might work. For instance in `jupyter` component we assume that jupyter is installed in the provided image.
- In some clusters the preset names specified in published actions (e.g. `cpu-small`) might not be available. What if we omit it and rely on `neuro-flow` for determining the preset name?

**Comment by YevheniiSemendiak** [comment]
> I suggest using curly brackets

@asvetlov, we decided to use the squared ones in the cookiecutter template to avoid conflicts with the underlying Jinja parser. Nevertheless, I have a question, is it OK for neuro-flow to mix bracket types within one yaml file (in one line we use `${{ one.thing }}`, while in the line below `$[[ other.thing ]]`? Or if we use squared ones in cookiecutter template, but curly brackets in hosted on GH actions?

File: {{cookiecutter.project_slug}}/.neuro/live.yml [hunk 2]
```diff
@@ -41,18 +41,13 @@ images:
...
-      PYTHONPATH: $[[ volumes.code.mount ]]
-    port_forward:
-      - "2211:22"
-    volumes:
-      - $[[ volumes.data.ref_ro ]]
-      - $[[ volumes.code.ref_rw ]]
-      - $[[ volumes.config.ref_rw ]]
-      - $[[ volumes.results.ref_rw ]]
-    cmd: bash
+    action: gh:neuro-actions/develop@master
```
[the remark this answers is not in the document]
Comment (anayden) [reply {{cookiecutter.project_slug}}/.neuro/live.yml:44]: Sounds good, will do!

File: {{cookiecutter.project_slug}}/.neuro/live.yml [hunk 3]
```diff
@@ -88,46 +83,23 @@ jobs:
...
+      multi_args: $[[ multi.args ]]
+      volumes_data_remote: $[[ volumes.data.remote ]]
+      volumes_code_remote: $[[ volumes.code.remote ]]
+      volumes_config_remote: $[[ volumes.config.remote ]]
+      volumes_notebooks_remote: $[[ volumes.notebooks.remote ]]
+      volumes_results_remote: $[[ volumes.results.remote ]]
 
   tensorboard:
+    action: gh:neuro-actions/tensorboard@master
     image: tensorflow/tensorflow:latest
```
[the remark this answers is not in the document]
Comment (anayden) [reply {{cookiecutter.project_slug}}/.neuro/live.yml:98]: Yes, thank you

**Comment by anayden** [comment]
> Is it possible to add small READMEs to those GH actions? At least one-sentence description and list of assumptions might work. For instance in jupyter component we assume that jupyter is installed in the provided image.

Yes, adding READMEs is in my plans.

> In some clusters the preset names specified in published actions (e.g. cpu-small) might not be available. What if we omit it and rely on neuro-flow for determining the preset name?

Do you mean completely omitting preset name? I'm not sure this will work as expected (e.g., when we need a GPU)

**Comment by asvetlov** [comment]
Sorry, I thought that it was an action's repository.
Yes, mixed brackets are allowed in the same yaml file.

**Review by YevheniiSemendiak — APPROVED** [review]
LGTM, thank you, @anayden !

**Pull request merged**